lm75bd/lm75bd.h

```diff
@@ -17,11 +19,19 @@
 #define LM75BD_OS_OP_MODE_COMP 0x00U
 #define LM75BD_OS_OP_MODE_INT 0x01U
 
+/* LM75BD Register Mappings (p.8) */ 
+#define LM75BD_POINTER_TEMP 0x00U   /* Temperature register (RO) */
+#define LM75BD_POINTER_CONF 0x01U   /* Configuration register (R/W) */
```

Comment:
since these macros are just used for the driver and don't need to be accessible to other modules, it would be better to put them in the .c file to limit their scope